### DISCUSSION TITLE
Is there a way to run this proxy as a docker container?

### DISCUSSION THREAD

**Discussion by devedse** [llamafilm/tesla-http-proxy-addon#147] [Q&A]
My Home Assistant installation is running as a docker container on my raspberry pi. Is there a way to run this addon as a separate container?

**Comment by iainbullock** [comment]
I have a project which does this

https://github.com/iainbullock/tesla-http-proxy-docker

**Reply by devedse** [reply]
I indeed found that just after posting this :smile:, still running into an issue with it though:
https://github.com/iainbullock/tesla-http-proxy-docker/issues/57

Let's move the discussion there

**Comment by iainbullock** [comment]
Referencing https://github.com/iainbullock/tesla-http-proxy-docker/issues/57

Issue resolved and closed
